book/tut2_spec_pd_gpd.ipynb

```diff
@@ -0,0 +1,961 @@
+      "text/plain": [
+       "<IPython.core.display.Image object>"
+      ]
+     },
+     "metadata": {
+      "image/png": {
+       "width": 500
+      }
+     },
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "# Choose the bin width of the bars; feel free to play around with this value\n",
+    "mag_min = df_jp_eqs[\"magnitude\"].min()\n",
+    "mag_max = df_jp_eqs[\"magnitude\"].max()\n",
+    "step_histo = 0.2\n",
+    "\n",
+    "fig = pygmt.Figure()\n",
+    "\n",
+    "fig.histogram(\n",
+    "    data=df_jp_eqs[\"magnitude\"],\n",
+    "    projection=\"X10c\",\n",
+    "    # Determine y range automatically\n",
+    "    region=[mag_min - step_histo, mag_max + step_histo * 2, 0, 0],\n",
+    "    # Define the frame, add labels to the x-axis and y-axis\n",
+    "    frame=[\"WSne\", \"x+lmagnitude\", \"y+lcounts\"],\n",
+    "    # Generate evenly spaced bins\n",
+    "    series=step_histo,\n",
+    "    # Fill bars with color \"orange\"\n",
+    "    fill=\"orange\",\n",
+    "    # Draw gray outlines with a width of 1 point\n",
+    "    pen=\"1p,gray\",\n",
+    "    # Choose histogram type 0, i.e., counts [Default]\n",
+    "    histtype=0,\n",
+    ")\n",
+    "\n",
+    "fig.show(dpi=150)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0658236e-869c-417c-8f4a-0195025e9430",
+   "metadata": {},
+   "source": [
+    "Use code example above as orientation, and create a similar histogram showing the hypocentral depth distribution of the earthquakes."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "743d7a80-c5d4-46ca-820c-451eae825aa6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Your code (:"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "b387494c-15ae-4019-83fc-bdf325f8a76d",
+   "metadata": {},
+   "source": [
+    "### 1.3 Create a geographical map of the earthquakes\n",
+    "\n",
+    "For plotting the earthquakes as size (magnitude) and color (depth) coded circles ontop of a map please follow the tutorial at\n",
+    "https://www.pygmt.org/v0.13.0/tutorials/basics/plot.html."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0b238bef-3018-4bdc-92f8-9caa888a2c32",
+   "metadata": {},
+   "source": [
+    "## 2. `GeoPandas`\n",
+    "\n",
+    "Different datasets are available from: https://github.com/GenericMappingTools/pygmt/issues/2786#issuecomment-1787655589"
```

Comment:
Maybe link to the geodatasets library docs at https://geodatasets.readthedocs.io/en/stable/introduction.html?

```suggestion
    "Different sample datasets are available from: https://geodatasets.readthedocs.io/en/stable/introduction.html"
```

Reply:
I moved this aspect to the "Additional comments" section, and added the link to the geodatasets library. Just used this link to GitHub as an reminder 🙂.